purchases/src/main/kotlin/com/revenuecat/purchases/common/networking/ETagManager.kt

```diff
@@ -12,14 +11,12 @@ import com.revenuecat.purchases.strings.NetworkStrings
 import org.json.JSONObject
 import java.util.Date
 
-@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-data class ETagData(
+internal data class ETagData(
     val eTag: String,
     val lastRefreshTime: Date?,
 )
 
-@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```

Comment:
Should we keep these annotations? They still should be considered private and are only `internal` in order to test them I think?

Reply:
You're correct yes